code/game/objects/structures/dropship_equipment.dm

```diff
@@ -762,7 +762,40 @@
 
 //////////////// OTHER EQUIPMENT /////////////////
 
+/obj/structure/dropship_equipment/operatingtable
+	name = "Dropship Operating Table Deployment System"
+	desc = "Used for advanced medical procedures. Fits on the crewserved weapon attach points of dropships. You need a powerloader to lift it."
+	equip_category = DROPSHIP_CREW_WEAPON
+	icon = 'icons/obj/surgery.dmi'
+	icon_state = "table2-idle"
+	point_cost = 300
+	var/obj/machinery/optable/deployed_table
+
+/obj/structure/dropship_equipment/operatingtable/Initialize()
+	. = ..()
+	if(!deployed_table)
+		deployed_table = new(src)
+
+/obj/structure/dropship_equipment/operatingtable/examine(mob/user)
+	. = ..()
+	if(!deployed_table)
+		to_chat(user, "Its table is broken.")
+		
+/obj/structure/dropship_equipment/operatingtable/Destroy()
+	QDEL_NULL(deployed_table)
+	return ..()
 
+/obj/structure/dropship_equipment/operatingtable/update_equipment()
+	if(!deployed_table)
+		return
+	if(ship_base)
+		deployed_table.layer = 3.03
```

Comment:
could this layer be a define?

Reply:
It could 
`#define ABOVE_OBJ_LAYER 3.02 //just above all items`
is what's used for all the dropship structures.

Please test if it is functional using this define, and add a new one at 3.03 if it doesn't.

Reply:
Changed it to 3.02, and it works fine.

Reply:
@Hathkar the point was less about the specific number but instead using a define.

If you check in `layers.dm` we define a ton of existing layers. `3.02` for example already exists there.

So instead of putting `layer = 3.03` consider `layer = ABOVE_OBJ_LAYER`

This is much clearer the intention and where the item should appear.